.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# Dependabot version updates raises a maximum of five pull requests each time it checks dependencies.
+# Note that there is some overlap with Dependabot security updates so some options can effect security updates as well,
+# see https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.
+
+version: 2
+updates:
+  # Maintain dependencies for npm ecosystem
+  - package-ecosystem: 'npm'
+    # Checks all directories from the current layer and below recursively for package.json files
+    directories:
+      - '**/*'
+    schedule:
+      # Runs every Monday
+      interval: 'weekly'
```

Comment:
why weekly and not say daily?

Reply:
I was thinking weekly to start so we aren't constantly bombarded with version update PRs, especially since we are using `increase-if-necessary` the PRs will most likely be for new MVs so we need to be thorough on checking for breaking changes.

Or we make sure E2E runs on these PRs and we assume if E2E passes then there are no breaks, in that case I'm fine with changing it to daily.